src/session/session.c

```diff
@@ -699,6 +711,13 @@ session_state_create (THREAD_ENTRY * thread_p, SESSION_ID * id)
    */
   ATOMIC_CAS_32 (&sessions.last_session_id, next_session_id, *id);
 
+  if (session_p->pl_session_p)
+    {
+      assert (false);
+      delete session_p->pl_session_p;
```

Comment:
```suggestion
      delete session_p->pl_session_p;
```

해당 조건이 실제로 만족될 가능성은 거의 없지만, 만약의 경우를 대비해 delete 대신 에러 메시지를 출력하는 편이 더 안전하다고 생각됩니다.

재현 테스트에서는 `session_p->pl_session_p` 에 임의의 값을 강제로 설정하면 조건을 만족시킬 수 있으며,
- Debug 모드에서는 assert가 발생하고
- Release 모드에서는 core dump가 발생하는 것을 확인했습니다.

이는 쓰레기 값으로 인해 delete가 호출될 경우, 소멸자(~session) 내부에서 core dump가 발생하기 때문입니다.

다만, 실제로는 이러한 상황이 발생하지 않는다고 판단됩니다.
그 이유는 `if (session_p->pl_session_p)` 조건에 도달하기 전에 항상 `session_state_init()` 내부에서 `session_p->pl_session_p` 가 NULL로 초기화되기 때문입니다.
```
session_state_create
-> (void) sessions.states_hashmap.insert()
...
-> session_state_init
```

재현 방법
1) cub_server를 attach
2) 아래 명령어 실행
```
$ csql -u dba demodb 
or 
csql> ;connect public
```
3) session_state_create() 함수의 `if (session_p->pl_session_p)` 조건(714 라인)에서 break를 걸고, session_p->pl_session_p 값을 임의의 값(예: 0x123)으로 수정 후 진행

Reply:
네. delete 는 garbage 값의 경우 위험 소지가 있고 (확률은 거의 없어보이지만)
ER_FAILED 를 리턴하는 것으로 바꾸었습니다. 
에러 메시지도 다시 포함시켰습니다.